src/main/resources/warnings.json

```diff
@@ -364,9 +364,11 @@
     "message": "Arbitrary code execution vulnerability",
     "url": "https://jenkins.io/security/advisory/2017-04-10/",
     "versions": [
-      {
-        "pattern": ".*"
-      }
+		{
+			"firstVersion": "0.1",
+			"lastVersion": "0.17",
+			"pattern": "\\b0\\.(\\d|1[1-7])\\b"
```

Comment:
This isn't matching substrings, so `\\b` are unnecessary.

For consistency, please use `[.]` for literal period chars.

We also generally try to allow for `1.2.3-whatever-3` version numbers, check other version regexes for how to allow for that.